src/query/api/v1/handler/prometheus/handleroptions/headers.go

```diff
@@ -28,6 +28,10 @@ import (
 )
 
 const (
+	// M3HeaderPrefix is the prefix all M3-specific headers that affect query or
+	// write behavior (not necessarily m3admin headers) are guaranteed to have.
+	M3HeaderPrefix = "M3-"
```

Comment:
Looks good - can you re-do all the values below to use it? e.g. `WarningsHeader = M3HeaderPrefix + "Warnings"`

If we have to change them to vars, that's fine - but would be best to do that to keep this standardized.